src/devices/src/virtio/vsock/packet.rs

```diff
@@ -93,6 +97,18 @@ pub struct VsockPacket {
     buf_size: usize,
 }
 
+fn get_host_address(
```

Comment:
Why don't we keep a single implementation of this function, bound to `GuestMemoryMmap`, keeping the `size` parameter in place?

Reply:
From what I know we have to declare a separate method, because this signature is not compatible with the method defined in `rust-vmm/vm-memory`, and it doesn't make sense to change the method signature in `vm-memory`.

Reply:
I see. Maybe another name for the `packet::get_host_address` would be more appropriate. I do not have a good suggestion for this. We can leave as it is for now, if there is no strong concern/preference for this.